### PR TITLE
[10.0.X] Update of MC misalignment scaler tool

### PR DESCRIPTION
**Pull request by ghellwig**
Addendum to #21264.

- add option to pull outlier modules to ideal position and orientation
- consistently scale also rotational misalignment
- used for MC scenario announced here: https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/3414.html

### CODE REVIEW

**Comment by ghellwig** [comment]
please test

File: Alignment/TrackerAlignment/test/mcMisalignmentScaler_cfg.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
     )
 )
 process.mcMisalignmentScaler.pullBadModulesToIdeal = False
+process.mcMisalignmentScaler.outlierPullToIdealCut = 0.1
```
Comment (arunhep) [inline Alignment/TrackerAlignment/test/mcMisalignmentScaler_cfg.py]: @ghellwig can you please add a comment here explaining the choice of this threshold?

Comment (arunhep) [reply Alignment/TrackerAlignment/test/mcMisalignmentScaler_cfg.py]: ping @ghellwig

Comment (ghellwig) [reply Alignment/TrackerAlignment/test/mcMisalignmentScaler_cfg.py]: @arunhep sorry, your question slipped through the cracks.
This cfg file is just an arbitrary example and as you can see above the default given in `fillDescriptions` is a negative value, i.e. outlier modules are *not* pulled to ideal.

**Comment by arunhep** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**